api_tests/files/views/test_file_metadata_record_detail.py

```diff
@@ -28,6 +30,10 @@ def preprint_record(user, preprint):
 @pytest.mark.django_db
 class TestFileMetadataRecordDetail:
 
+    @pytest.fixture(autouse=True)
+    def datacite_file_schema(self):
+        return ensure_datacite_file_schema()
```

Comment:
Can we move this fixture out of the class so that we only define / run it once in [test_file_metadata_record_detail.py](https://github.com/CenterForOpenScience/osf.io/pull/9994/files#diff-dda3946d9654d5f434661645cefa7d9abd8dddd337ad1161d0ad6d1528c72310)?